[2010-03-23 21:41:30,134] INFO     0[main](AssertMain.java:17) - Results asserter starting
[2010-03-23 21:41:30,137]DEBUG     3[main](AssertMain.java:20) - Program arguments:
[2010-03-23 21:41:30,142]DEBUG     8[main](AssertMain.java:22) - args[0] 'throw-soapfaultexception-cout'
[2010-03-23 21:41:30,142]DEBUG     8[main](AssertMain.java:22) - args[1] 'D:\dev\stepframework\src_test\extensions-testbed\results\20100323\2135\throw-soapfaultexception-cout'
[2010-03-23 21:41:30,142]DEBUG     8[main](AssertMain.java:45) - Building context
[2010-03-23 21:41:30,158]TRACE    24[main](AssertMain.java:66) - Context:
[2010-03-23 21:41:30,159]TRACE    25[main](AssertMain.java:68) - resultDir: D:\dev\stepframework\src_test\extensions-testbed\results\20100323\2135\throw-soapfaultexception-cout
[2010-03-23 21:41:30,159]TRACE    25[main](AssertMain.java:68) - clientSoap: D:\dev\stepframework\src_test\extensions-testbed\results\20100323\2135\throw-soapfaultexception-cout\soap-trace-client.log
[2010-03-23 21:41:30,160]TRACE    26[main](AssertMain.java:68) - serverLog: D:\dev\stepframework\src_test\extensions-testbed\results\20100323\2135\throw-soapfaultexception-cout\log-server.log
[2010-03-23 21:41:30,160]TRACE    26[main](AssertMain.java:68) - config: throw-soapfaultexception-cout
[2010-03-23 21:41:30,160]TRACE    26[main](AssertMain.java:68) - serverSoap: D:\dev\stepframework\src_test\extensions-testbed\results\20100323\2135\throw-soapfaultexception-cout\soap-trace-server.log
[2010-03-23 21:41:30,160]TRACE    26[main](AssertMain.java:68) - clientLog: D:\dev\stepframework\src_test\extensions-testbed\results\20100323\2135\throw-soapfaultexception-cout\log-client.log
[2010-03-23 21:41:30,161]DEBUG    27[main](AssertMain.java:76) - Entering assert dispatch
[2010-03-23 21:41:30,161]DEBUG    27[main](AssertMain.java:79) - Asserting configuration 'throw-soapfaultexception-cout'
[2010-03-23 21:41:30,179] INFO    45[main](Assertions.java:52) - assertClientSOAPFaultException
[2010-03-23 21:41:30,199] INFO    65[main](Assertions.java:162) - assertSOAPCounts
[2010-03-23 21:41:30,200] INFO    66[main](Assertions.java:118) - assertCountClientSOAPEnvelopes
[2010-03-23 21:41:30,200] INFO    66[main](Assertions.java:123) - assertCountServerSOAPEnvelopes
[2010-03-23 21:41:30,218] INFO    84[main](Assertions.java:128) - assertCountClientSOAPFaults
[2010-03-23 21:41:30,219] INFO    85[main](Assertions.java:133) - assertCountServerSOAPFaults
[2010-03-23 21:41:30,235] INFO   101[main](AssertMain.java:234) - SUCCESS! All assertions executed successfully for configuration 'throw-soapfaultexception-cout'
[2010-03-23 21:41:30,235] INFO   101[main](AssertMain.java:37) - End of results asserter.
